site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -312,6 +313,33 @@ If managed FusionAuth hosting does not meet your needs, open a link:/contact/[co
 
 For instance, if you need longer retention of database backups, we're happy to discuss that with you.
 
+== SLAs and Availability
+
+We know that availability of your auth system is critical. Below are some of the steps FusionAuth takes to ensure that FusionAuth cloud systems are available. You can find information about service level agreements (SLAs) in the link:/license/[FusionAuth license] and link:/license-faq/[license FAQ].
+
+For all deployments:
+
+* All components are monitored via external software systems as well as internal metrics gathering systems.
+* The application architecture is not exotic, reducing risk. This is a three tier web application, which is a well understood architecture.
+* External security researchers are encouraged to submit security issues via a bug bounty program.
```

Comment:
External or 3rd Party? Any preference? 

Encouraged, and do..  not sure if we want to phrase it that way or not.

Reply:
Tweaked wording if you want to take a look.